tests/testlogout.js

```diff
@@ -0,0 +1,31 @@
+//const { default: $ } = require('webdriverio/build/commands/browser/$');
```

Comment:
Remove this file, as it's not part of the Login related logic, which is the focus of this Pull Request